docs/Behaviour - Client Side.md

```diff
@@ -13,14 +13,16 @@ Constraint Sets of Active Constraints SHOULD represent a consensus among Receive
 Constraint Sets of Receiver Capabilities with `urn:x-nmos:cap:meta:enabled` set to false MUST be ignored completely while making the processing.
 
 
-## Dynamic format changes on Sender
+## Dynamic Update of Resources
 
-### Changes which break Active Constraints
+The Controller MUST be capable of using Stream Compatibility API in conjunction with IS-04 APIs to discover and dynamically report the state of any relevant resources, including Inputs, Outputs, Senders and Receivers.
```

Comment:
```suggestion
The Controller uses Stream Compatibility API in conjunction with IS-04 APIs to discover and track the state of any relevant resources, including Inputs, Outputs, Senders and Receivers.
```

Reply:
Should it be more specific "The Controller uses Stream Compatibility API in conjunction with IS-04 Query API to discover and track the state of any relevant resources, including Inputs, Outputs, Senders and Receivers." because IS-04 says "The Controller uses the Query API to discover and track the state of any registered resource, including Node, Device, Source, Flow, Sender, and Receiver."

Reply:
There is no Query API in the Peer to Peer mode and I tried to include this case. I will raise it in the IS-04 PR.